### DISCUSSION TITLE
Search does not work!

### DISCUSSION THREAD

**Discussion by K69G4JRA725W** [requarks/wiki#3714] [Error / Bug Report]
**Describe the bug**
Search Functionality does not work for Chinese contents!

**To Reproduce**
1. Go to 'http://wiki.medsino.net'
2. Focus the mouse pointer on 'search box'
3. Input '麻黄汤‘ or 'Categories',  and hit  'Enter' key.

**Expected behavior**
It should list the page under the tag '麻黄汤‘, or the page that contains the word 'Categories'.

**Host Info (please complete the following information):**
 - OS: CENT-OS 7
 - Wiki.js version:  2.0.12
 - Database engine: postgres 12

**Additional context**
1. Search by Tag and Full text both does not work!
2. In the Admin section, I chose Database - PostgreSQL. In the 'Dictionary Language' dropdown, there is no option for Chines. Is this the source of  the problem for search?

**Comment by NGPixel** [comment]
The dictionary language is a list provided by the Postgres engine. If your language is not listed, you should use either simple or English.

I'll investigate your other issue.

**Comment by kingreatwill** [comment]
@NGPixel Can custom dictionary language be supported.

**Comment by NGPixel** [comment]
@kingreatwill Submit a feature request for it.

**Comment by kingreatwill** [comment]
https://wiki.js.org/feedback/p/can-custom-dictionary-language-be-supported

**Comment by zbjdonald** [comment]
You can install zhparser pg extension to support chinese full text search. See https://zhuanlan.zhihu.com/p/335359081.
